test/e2e/setup-kubeflex.sh

```diff
@@ -46,7 +48,19 @@ make ko-local-build
 :
 :
 make install-local-chart
+:
+:
+else
+  echo "Installing kubeflex release: ${KUBEFLEX_RELEASE}"
+
+  helm repo add kubeflex https://kubestellar.github.io/kubeflex
+  helm repo update
 
+  helm install kubeflex kubeflex/kubeflex \
+    --namespace kubeflex-system \
+    --create-namespace \
+    --version "${KUBEFLEX_RELEASE}"
```

Comment:
It would be better if this aligned more closely with how ks/ks uses the KubeFlex chart. E.g., see https://github.com/kubestellar/kubestellar/blob/9062484ec872310e351bb578a7a257229a979239/core-chart/Chart.yaml#L31-L34 and https://github.com/kubestellar/kubestellar/blob/9062484ec872310e351bb578a7a257229a979239/docs/content/direct/get-started.md#use-core-helm-chart-to-initialize-kubeflex-and-create-its-and-wds . Note that these are instantiating the charts into the default namespace.